proto/api_v2/metricsquery.proto

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax="proto3";
+
+package jaeger.api_v2;
+
+import "model.proto";
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+import "opentelemetry/proto/metrics/v1/metrics.proto";
+
+option go_package = "api_v2";
+option java_package = "io.jaegertracing.api_v2";
+
```

Comment:
I could not include the `marshalto`, etc. [Gogo proto extensions](https://github.com/gogo/protobuf/blob/master/extensions.md) as [query.proto does](https://github.com/jaegertracing/jaeger-idl/blob/master/proto/api_v2/query.proto#L29-L35) because the OTEL metrics doesn't adopt them, leading to incompatibilities.